pom.xml

```diff
@@ -455,6 +492,7 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-compiler-plugin</artifactId>
+                <version>3.8.0</version>
```

Comment:
Any reason this is 3.8.0 and not 3.8.1?

Reply:
Most likely because I either didn't realize 3.8.1 was released, or at the time I made the original update to use 3.8.0 to PR 3404, that was the latest version of the plugin.  Either way, I can update this PR to use 3.8.1.